app/models/purchase.rb

```diff
@@ -3201,7 +3204,9 @@ def calculate_additional_discover_fee_per_thousand
     end
 
     def calculate_gumroad_fee_per_thousand
-      if flat_fee_applicable?
+      if custom_direct_fee_percentage.present?
+        (custom_direct_fee_percentage.to_f * 10) # convert 10% → 100 per 1000
```

Comment:
```suggestion
        (custom_direct_fee_percentage.to_f * 10).round + (charged_using_gumroad_merchant_account? ? PROCESSOR_FEE_PER_THOUSAND : 0)
```
Current changes include the 30c fixed part of the processor fee but not the 2.9% variable part, this adds the 2.9% variable processor fee as well.